### ISSUE TITLE
Duplicate values in CASE statement (when utf-8)

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1101]
Our cpstream / cpinternal is utf-8:
````
DEFINE VARIABLE cc AS CHARACTER NO-UNDO.

cc = 'Ï':u.

CASE cc:
   WHEN 'Í':u THEN MESSAGE 1 cc.
   WHEN 'Ï':u THEN MESSAGE 2 cc.
END CASE.
````

False positive with incorrect message, indicating that there are some codepage shenannigans going on:

![image](https://github.com/Riverside-Software/sonar-openedge/assets/5297464/40266bb1-6f03-49e7-8846-c12a7cd564a3)

Interestingly, this issue only manifests in PDSOE, the SonarQube instance does not have this issue.

**Comment by gquerret** [comment]
Reproduced. Will be fixed in next sonarlint-eclipse release.

**Comment by gquerret** [comment]
Fixed in develop branch of sonar-openedge (no need to update sonarlint-eclipse).
